packages/lit-dev-content/site/_includes/api-v1.html

```diff
@@ -0,0 +1,319 @@
+---
```

Comment:
This is a copy of the `api.html` layout (so v1 docs that never change again stay in sync with their layout that never changes). And we can still update newer docs with newer layouts.